src/execution/sema/sema_builtin.cpp

```diff
@@ -3059,6 +3080,8 @@ void Sema::CheckBuiltinCall(ast::CallExpr *call) {
       break;
     }
     case ast::Builtin::StorageInterfaceInit:
+    case ast::Builtin::InitTablePR:
```

Comment:
Is there a reason you're using table PRs instead of just getting it directly from the VPI?

`@prSetIntNull(update_pr, 0, @vpiGetIntNull(vpi, 0))`
vs
`@prSetIntNull(index_pr, 0, @prGetIntNull(table_pr, 0))`

If we go with the table_pr approach, this requires an extra allocation and deallocation for the table_pr with every storage interface instantiation.

Reply:
Now it follows @prSetIntNull(update_pr, 0, @vpiGetIntNull(vpi, 0))